setup.py

```diff
@@ -36,6 +36,7 @@
         "markdown-link-attr-modifier==0.2.1",
         "mdx_truly_sane_lists==1.2",
         "openpyxl~=3.0.3",  # this package is needed for script only https://github.com/DOAJ/doajPM/issues/2433
+        "pandas==2.0.1"
```

Comment:
If we're just using this in the script, perhaps it makes sense to add it to the `extras_requre` section (like `test` or `docs`) and install with `pip install -e .[urls]` so we're not installing such a heavyweight library (and risking conflicts) in production